IPyth.sol

```diff
@@ -74,6 +74,26 @@ interface IPyth {
     /// @param updateData Array of price update data.
     function updatePriceFeeds(bytes[] memory updateData) external payable;
 
+    /// @notice Wrapper around updatePriceFeeds that rejects fast if a price update is not needed based on the
+    /// sender known latest publish time of prices that it wants to update. This method rejects solely based on
+    /// given `publishTimes` for the price feeds. It means that the prices has been updated since the sender
+    /// observed the price latest publish time. However, `updateData` might still be a newer update. If you
+    /// want to update the contract price to the latest available price, please use `updatePriceFeeds` and
+    /// do not call this method.
```

Comment:
this part of the documentation is confusing -- how can updateData contain a newer update if the publishTime is older?

Reply:
I explained this one in a thread with Tom. Please look at that. I will modify the docs to make it more clean.